packages/discovery-provider/ddl/migrations/0107_add_email_tables.sql

```diff
@@ -0,0 +1,46 @@
+begin;
+
+-- Store encrypted emails with their metadata
+CREATE TABLE IF NOT EXISTS encrypted_emails (
+    email_id INTEGER PRIMARY KEY,
```

Comment:
Are we explicitly not wanting to link emails to `user_id`?

Reply:
good catch, was waiting to confirm in product sync, but we do! adding